controls/pcidss_4_ocp4.yml

```diff
@@ -861,9 +859,46 @@ controls:
           access to unencrypted data are stored securely.
         levels:
           - base
-        status: pending
+        status: partial
         notes: |-
-          To properly check this requirement, site policies and documentation should be consulted.
+          Disk-level encryption is a node configuration, but it can be managed by the NBDE
+          Tang Server Operator to automatically unlock LUKS-encrypted volumes and rotate keys.
+
+          In case where card holder data may be stored on worker nodes disks, OpenShift Container
+          Platform can help to partially
+          comply with this requirement by providing several means of fully encrypting disks.
+          * Using RHEL core crypto components which are FIPS certified
+          * Or using cloud provider techniques, such as EBS encryption for AWS case.
+          For this requirement, we also check etcd is encrypted, so that secrets, but also routes
+          and oauth configurations are secured.
+
+          Concerning Full Disk Encryption:
+          Full Disk Encryption can be enabled on OpenShift by installing the cluster with FIPS mode
+          enabled.
+          OpenShift Container Platform uses certain FIPS Validated / Modules in Process modules
+          within RHEL and RHCOS for the operating system components that it uses.
+          See RHEL7 core crypto components and https://docs.openshift.com/container-platform/4.15/installing/installing-fips.html
```

Comment:
Even though this is a note, and not exposed to end users, we may want to consider removing the versions that will become obsolete.

https://docs.openshift.com/container-platform/latest/installing/installing-fips.html - will reference 4.15 today and 4.16 when it becomes available.

Also - the 4.15 page lists RHEL8 instead of RHEL7?